WhaleSpotting/react-app/src/api/apiClient.ts

```diff
@@ -0,0 +1,31 @@
+import authService from "../components/api-authorization/AuthorizeService";
+
+export async function makeAdmin() {
+    const token = await authService.getAccessToken();
+    const response = await fetch("User/MakeAdmin", {
+        headers: !token ? {} : { "Authorization": `Bearer ${token}` }
+    });
```

Comment:
Can we make the creation of this headers object a function down below, so we just have 
```suggestion
    const headers = await getHeaders();
    const response = await fetch("User/MakeAdmin", {
        headers: headers
    });
```